### PR TITLE
docs: generate release note and links

### PR DESCRIPTION
**Pull request by caspahouzer**

### CODE REVIEW

File: docs/guide/Titanium_SDK/Titanium_SDK_Release_Notes/README.md [hunk 1]
```diff
@@ -181,6 +180,9 @@ Titanium SDK 11.0.0 is a major release of the SDK, under new leadership of TiDev
 
 * textarea contentHeight fix with new return property textareaHeight in "change" event ([94820d1](https://github.com/tidev/titanium_mobile/commit/94820d1952d5e301b6ea2741576b3046b5b382b5))
 
+## BREAKING CHANGES
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_Release_Notes/README.md:183]: Why list breaking changes if there are none?

Comment (caspahouzer) [reply docs/guide/Titanium_SDK/Titanium_SDK_Release_Notes/README.md:183]: Fixed

**Comment by hansemannn** [comment]
Fixed the date in one of it's occurrences. Should we add a note that this one can already be used to test Xcode 15 Beta and iOS 16 Beta? May be a good reason for users to upgrade sooner.

**Review by cb1kenobi — APPROVED** [review]
**APPROVED**!

**Comment by cb1kenobi** [comment]
@hansemannn I love the idea of stating 11 GA can be used with iOS and Xcode betas. I never understood the whole day 1 support thing when we should support the betas and give people time to upgrade their apps so they can actually ship on the day iOS drops.

**Comment by cb1kenobi** [comment]
Please wait to merge this until @joshualambert and friends have finished the blog post!!!!

**Pull request merged**